reconcile/prometheus_rules_validator.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import semver
+import logging
+
+import utils.gql as gql
+import utils.threaded as threaded
+import utils.promtool as promtool
+import reconcile.openshift_resources_base as orb
+import reconcile.queries as queries
+
+QONTRACT_INTEGRATION = 'prometheus_rules_validator'
+QONTRACT_INTEGRATION_VERSION = semver.format_version(0, 1, 0)
+
+
+def check_rule(rule):
+    try:
+        promtool.check_rule(rule['spec'])
+    except Exception as e:
+        return {'path': rule['path'], 'message': str(e).replace('\n', '')}
```

Comment:
why to wrap `utils.promtool.check_rule` here? can't it give you the right thing to begin with?

Reply:
What do you suggest, that instead of an exception on error it returns something I can inspect, e.g. an object?

Reply:
You know me... I'd return a custom object from `utils.promtool.check_rule()` tha implements `__bool__` and carries a message :)
But a dict or a custom object with two attributes (status and message) should do it.

Reply:
fixed in eb8e705. I loved the fancy approach, I hope I implemented it properly

Reply:
looks awesome :)